client/src/inside/components/mar/mar-today.js

```diff
@@ -83,7 +85,10 @@ export default class MarToday {
 
     // Match mars for the current data to the scheduled list of medications
     marRecsFiltered.forEach( mar =>  {
-      if(mar.scheduledTime ) {
+      const pk = aDaySchedule.find( pk => 
+        pk.hour24 === mar.scheduledTime
+      )
+      if(mar.scheduledTime && pk) {
```

Comment:
Please add a comment to explain what's happening here.

Reply:
This is related to the issue's step one.